app/src/main/java/com/gst/gusto/repository/AuthRepositoryImpl.kt

```diff
@@ -0,0 +1,21 @@
+package com.gst.gusto.repository
+
+import com.gst.gusto.api.ApiResponse
+import com.gst.gusto.datasource.AuthDataSource
+import com.gst.gusto.dto.ResponseRefreshToken
+import okhttp3.ResponseBody
+
+class AuthRepositoryImpl(
+    private val authDataSource: AuthDataSource
+) : AuthRepository {
+    override suspend fun getRefreshToken(
+        xAuthToken: String,
+        refreshToken: String
+    ): ApiResponse<ResponseRefreshToken> {
+        return when (val response = authDataSource.getRefreshToken(xAuthToken, refreshToken)) {
+            is ApiResponse.Success -> ApiResponse.Success(response.data)
+            is ApiResponse.Error -> ApiResponse.Error(response.errorCode, response.errorMessage)
+        }
```

Comment:
이미 `getRefreshToken()` 가 ApiResponse 를 반환하고 있는 것 같은데요, 요거 그대로 리턴하면 안되나요 ? 
```suggestion
        return authDataSource.getRefreshToken(xAuthToken, refreshToken)
```